package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
-  name: 'nimble:restivus',
+  name: 'maka:rest',
```

Comment:
I'm not sure if is a good idea to rename the package.
May you could close this PR and open a new one without those commits, what you think?